examples/server/lwm2mserver.c

```diff
@@ -321,6 +401,13 @@ static void prv_discover_client(lwm2m_context_t *lwm2mH, char *buffer, void *use
     fprintf(stdout, "Syntax error !");
 }
 
+/**
+ * @brief Writes data to a client.
```

Comment:
What are `LWM2M_CONTENT_JSON`,`LWM2M_SUPPORT_JSON`,`LWM2M_CONTENT_TEXT` definitions?
Of course that we are not going to document self-explanatory definitions as e.g. `COAP_500_INTERNAL_SERVER_ERROR`, but other ones are needed.

Reply:
These are defined and mentioned in readme file. Check there.